data/wanderer/wanderers start.txt

```diff
@@ -1856,10 +1865,11 @@ mission "Wanderers Hai Assistance 2"
 	on offer
 		log "Traveled to the Hai homeworld with Ambassador Sayari to see if she can convince the Hai to take action to restrain the Unfettered from attacking the Wanderers. The most that the Hai were willing to do was to mass a large war fleet near Unfettered space so that the Unfettered have to divert warships there to defend their home worlds."
 		conversation
-			`The Hai are ruled by an elected "council of elders" who agree to meet with you and Sayari. After you describe the situation with the Wanderers, one of the elders says, "We had heard rumors of this. The Unfettered have recruited thousands of new warriors by telling stories of planets for us to explore and colonize, and a cowardly and foolish enemy to defeat."`
+			`The Hai's recently elected "council of elders" agrees to meet with you and Sayari and the two of you make an effort to avoid directing yourselves towards Teeneep. Your friend's popularity as an Elder does not extend to inside these walls, and it would be detrimental to your cause to seek her assent specifically here. After you describe the situation with the Wanderers, one of the other elders says, "We had heard rumors of this. The Unfettered have recruited thousands of new warriors by telling stories of planets for us to explore and colonize, and a cowardly and foolish enemy to defeat."`
 			`	Sayari says, "We must not allow the Unfettered to commit genocide in the name of the Hai people."`
 			`	Another elder responds, "But, we promised in the Frostmark Accords that we would not interfere in Unfettered space."`
 			`	"They have never kept their side of the treaty," says yet another of the elders, "and surely the prevention of genocide is enough reason to break ours."`
+			`	Here Teeneep makes an obvious noise of disgust, which draws a few mixed looks but is not commented on.`
```

Comment:
This reaction from Teeneep is not appropriate to her character at all nor consistent with the rest of the story.

Teeneep is not an emotional person prone to outbursts. She is cold and calculating and invents a personality as needed to manipulate people. Notice how she acted almost like a different person around the Unfettered, the human heads of state, and the Hai kidnapping victims. She manipulated the entire situation surrounding the kidnappings to propel herself to the Hai Elder Council and a national hero on both sides of the Hai border. She would have spent the last few months gaining power in the Elder Council. 

As the recognized expert on all things Unfettered, I would expect her to lead this meeting, not be a disrespected member on the sidelines. She would use this to advance her plans of Hai unification (through war or peace as needed) without letting the other Elders realize that's what she's doing. As the Unfettered are vulnerable right now, she wouldn't push for an invasion. It would be self-destructive to the Hai as a whole. I would expect her to be the one to suggest moving a fleet to the border.

I suggest this,

> Teeneep responds, "You can hardly fault them for that; a treaty under duress is no treaty at all. No amount of begging or negotiating will convince them to stop their war. They recognize only strength, not the pleas of a coward. We must show strength, and so must the Wanderers. We'll increase our fleets at Wah Ki, making ourselves more of a threat."
> The other Elders nod in agreement.

Reply:
If you want to take a pass at this, please do so. Otherwise I'll work on it after I fix the mission coding bugs.

Reply:
A pass has been taken.